Sources/Quick/Configuration/QuickConfiguration.swift

```diff
@@ -25,10 +25,11 @@ internal func qck_enumerateSubclasses<T: AnyObject>(_ klass: T.Type, block: (T.T
     var subclass, superclass: AnyClass!
     for i in 0..<classesCount {
         subclass = classes[Int(i)]
-        superclass = class_getSuperclass(subclass)
-        if superclass === klass {
-            block(subclass as! T.Type) // swiftlint:disable:this force_cast
-        }
+
+        guard let superclass = class_getSuperclass(subclass),
+            superclass === klass else { return }
```

Comment:
The `return` here would halt enumeration of the classes defined in this bundle, no? Should this be a `continue` instead? Let me know if I'm misreading this 😅

Reply:
Yeah, you are right, sorry about that. I've just made it a simple `if` like it was before.